src/background/proxyHandler/proxyHandler.js

```diff
@@ -108,9 +112,27 @@ export class ProxyHandler extends Component {
    */
   processClientStateChanges(vpnState) {
     console.log(`Processing client state change ${vpnState}`);
-    this.#mLocalProxyInfo.value = vpnState.loophole
-      ? [ProxyUtils.parseProxy(vpnState.loophole)]
-      : [];
+    if (this.#mPlatformOs === "linux") {
```

Comment:
This can be set when we get the plattformOs, no need to do that everytime we get a vpn state (which is often)